public/app/features/alerting/unified/hooks/useCombinedRuleNamespaces.ts

```diff
@@ -275,11 +275,11 @@ export function calculateRuleTotals(rule: Pick<AlertingRule, 'alerts' | 'totals'
   }
 
   return {
-    alerting: result[AlertInstanceTotalState.Alerting],
+    alerting: result[AlertInstanceTotalState.Alerting] || result['firing'],
     pending: result[AlertInstanceTotalState.Pending],
     inactive: result[AlertInstanceTotalState.Normal],
     nodata: result[AlertInstanceTotalState.NoData],
-    error: result[AlertInstanceTotalState.Error] + result['err'], // Prometheus uses "err" instead of "error"
+    error: result[AlertInstanceTotalState.Error] + result['err'] || undefined, // Prometheus uses "err" instead of "error"
```

Comment:
I think we should set [AlertInstanceTotalState.Error] ?? result['err'] ??? as we are not going to have both at the same time.